api/src/pcapi/core/bookings/api.py

```diff
@@ -298,6 +301,22 @@ def book_offer(
     )
     amplitude_events.track_book_offer_event(booking)
 
+    provider = providers_repository.get_provider_enabled_for_pro_by_id(stock.offer.lastProviderId)
+    if (
+        not booking.stock.offer.withdrawalType == offers_models.WithdrawalTypeEnum.IN_APP
```

Comment:
[détail / style] Vu la longueur de la ligne, le temps d'arriver au "==", on aura oublié le "not". ;) Autant utiliser "!=".

```suggestion
        booking.stock.offer.withdrawalType != offers_models.WithdrawalTypeEnum.IN_APP
```